src/ExtensionGallery/.vscode/launch.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
I accidentally committed my .vscode directory without thinking.  These can definitely be excluded if desired.